angular.json

```diff
@@ -16,7 +16,7 @@
             "outputPath": "dist",
             "index": "src/index.html",
             "main": "src/main.ts",
-            "tsConfig": "src/tsconfig.app.json",
+            "tsConfig": "src/tsconfig.ee.json",
```

Comment:
We can use `src/tsconfig.ee.json` by default. Additionally, in the development there is no need to exclude any sources. It has to be done for builds only.